plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -93,6 +93,8 @@ public function __construct() {
 		remove_action( 'admin_print_scripts', 'print_emoji_detection_script' );
 
 		add_action( 'admin_init', array( __CLASS__, 'deactivate_wc_admin_plugin' ) );
+
+		add_action( 'in_admin_footer', array( __CLASS__, 'add_appearance_theme_view_tracks_event' ) );
```

Comment:
I used `in_admin_footer` because there doesn't seem to be a direct hook in the themes screen we could use.

Reply:
I've asked ChatGPT and an alternative is we could use `add_action( 'load-themes.php', array( __CLASS__, 'add_appearance_theme_view_tracks_event' ) );`. With this, we don't need to check for the current screen as the hook is only executed when it's in the themes page.

This is based on this general hook: https://developer.wordpress.org/reference/hooks/load-pagenow/

Reply:
Thanks @ilyasfoo ! I've updated it in [b29ad62](https://github.com/woocommerce/woocommerce/pull/40193/commits/b29ad629a874ae0eb8e1996938ce754536e0b574) and it seems to work well!